src/blog/2024/07/deploying-flowfuse-with-docker.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Deploying FlowFuse with Docker on an Ubuntu server
+subtitle: Step-by-step guide on how to deploy FlowFuse with Docker on Ubuntu server
+description: Learn to deploy the FlowFuse on your Ubuntu server with Docker, from domain setup to running your app with email, SSL configurations.
+date: 2024-07-08
+authors: ["sumit-shinde"]
+image:
+tags:
+ - node-red
+ - flowfuse
+ - self hosted
+ - docker
+---
+
+With Node-RED's increasing role in IoT, FlowFuse Cloud has become a favored platform for deploying production Node-RED applications. It offers [extensive features](/product/features/) at a low cost, reducing operational overhead. However, the cloud is not the only option we provide; we also offer a self-hosted option for users who prefer to deploy FlowFuse on their servers. This guide demonstrates how to deploy FlowFuse on your Ubuntu server using Docker, covering key aspects such as domain setup, email, SSL, and more for real-world production scenarios
+
+<!--more-->
+
+## What is Docker?
+
+[Docker](https://docs.docker.com/guides/docker-concepts/the-basics/what-is-an-image/) is an open-source platform that simplifies how applications are deployed, scaled, and managed through containerization. It enables you to package all components required by your project such as code, libraries, and dependencies into a single, portable unit known as a [Docker container](https://docs.docker.com/guides/docker-concepts/the-basics/what-is-a-container/). These containers ensure consistency in application environments and ease deployment by ensuring that applications run predictably across different computing environments, whether on a developer's laptop, a server, or a cloud platform.
+
+## Deploying FlowFuse on Ubuntu server with Docker
+
+Before proceeding, ensure you have your domain and a server with Ubuntu installed. If your server has Ubuntu installed without a GUI, you can connect to it from your computer using [SSH](https://itsfoss.com/set-up-ssh-ubuntu/), so that you can run commands from your local computer on the server.
+
+### Installing Docker on the Ubuntu server.
```

Comment:
Should we have documentation on our website on how to install other company's products?  Shouldn't this be a reference?

Reply:
This is in reference to Docker.

Reply:
Removed that section and added a link to their docs.